app/build.gradle

```diff
@@ -61,4 +61,6 @@ dependencies {
 
 
 
+
+
```

Comment:
If there's no changes, revert the file.
you can do it in terminal 
`git checkout HEAD -- <my-file.txt>`